torchvision/transforms/functional_tensor.py

```diff
@@ -1179,3 +1179,30 @@ def gaussian_blur(img: Tensor, kernel_size: List[int], sigma: List[float]) -> Te
 
     img = _cast_squeeze_out(img, need_cast, need_squeeze, out_dtype)
     return img
+
+
+def invert(img: Tensor) -> Tensor:
+    """PRIVATE METHOD. Invert the colors of a grayscale or RGB image.
+
+    .. warning::``
+
+        Module ``transforms.functional_tensor`` is private and should not be used in user application.
+        Please, consider instead using methods from `transforms.functional` module.
+
+    Args:
+        img (Tensor): Image to have its colors inverted in the form [C, H, W].
+
+    Returns:
+        Tensor: Color inverted image Tensor.
+    """
+    if not _is_tensor_a_torch_image(img):
+        raise TypeError('tensor is not a torch image.')
+
+    if img.ndim < 3:
+        raise TypeError("Input image tensor should have at least 3 dimensions, but found {}".format(img.ndim))
+
+    _assert_channels(img, [1, 3])
+
+    max_val = _max_value(img.dtype)
```

Comment:
If image dtype is float then max_val will be something very large right ? OK, probably, now I understand the questions asked about float dtype and its range as 0-1.
How about adding an option to the method such that user could specify its own max value. By default can be max dtype...

Reply:
Yep you are right. I opted for using the same approach as _bend() to determine the `bound`/`max_value`. I did not optin for passing it as a param to make sure the API is similar as other transformations.

Reply:
Hum, this sounds fishy to me for floating point types. We should probably not be doing `1.79769e+308 - value` for doubles, nor `3.40282e+38 - value` for floats.

Is there any use-case for applying color transforms to native float images (like satellite  or medical data)? If not, we should probably assume that float images are within 0-1 range

Reply:
For medical imagery I'd use MONAI as reference and it seems like it is possible: https://docs.monai.io/en/latest/transforms.html#intensity . They do not have any assumptions on the range of input data (uint8 or float).

Reply:
That's a great reference, thanks @vfdev-5 !

I think that exposing a `min_value / max_value` arguments (which defaults to `None` and has sensible defaults for different data types) is a good trade-off to consider.

We can go in two steps though if we want to be extra cautious, and first raise an exception and then in the future add those extra arguments.

Thoughts?